bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
+    embedding_vector_size = params['emb1_size']
+    embeddings = []
+    for i in range(len(w2i)):
+        term_emb = None
+        if i in i2w:
+            regular_term = str(i2w[i])
+            capitalized_term = regular_term.capitalize()
+            if regular_term in mdl:
+                term_emb = mdl[regular_term]
+            elif capitalized_term in mdl:
+                term_emb = mdl[capitalized_term]
+
+        if term_emb == None:
+            term_emb = np.zeros(embedding_vector_size,)
+
+        embeddings.append(term_emb)
+    return np.array(embeddings)
+
+def create_vocab_output_files(emb_i, i2w, zero_vectors_out_file, nonzero_vectors_out_file):
```

Comment:
Good point, but then please add a doctstring that explains, what the function does